spec/Vehicle/Vehicle.vspec

```diff
@@ -184,16 +184,43 @@ Speed:
   description: Vehicle speed.
 
 TravelledDistance:
+  deprecation: V3.1 moved to Vehicle.TraveledDistance
   datatype: float
   type: sensor
   unit: km
-  description: Odometer reading, total distance travelled during the lifetime of the vehicle.
+  description: Odometer reading, total distance traveled during the lifetime of the vehicle.
 
-TripMeterReading:
+TraveledDistance:
   datatype: float
   type: sensor
   unit: km
-  description: Current trip meter reading.
+  description: Odometer reading, total distance traveled during the lifetime of the vehicle.
+
+TraveledDistanceSinceStart:
+  datatype: float
+  type: sensor
+  unit: km
+  description: Distance traveled since start of current trip.
+  comment: A new trip is considered to start when engine gets enabled (e.g. LowVoltageSystemState in ON or START mode).
+           A trip is considered to end when engine is no longer enabled.
+           The signal may however keep the value of the last trip until a new trip is started.
+
+TimeSinceStart:
```

Comment:
Proposing to move this signal from FuelSystem to Vehicle as it might be relevant for any vehicle

Reply:
+1

Reply:
Meeting notes:
Daniel: Would it be better to have starttime than time since start?
Ulf: Dont we not need a signal for "now" as well
Daniel: Assume apps can always get current time
Erik: Have no problems having start time instead